src/symon/index.ts

```diff
@@ -296,49 +296,59 @@ export default class SymonClient {
     }
   }
 
-  private willSendEventListener({
+  private async willSendEventListener({
     probeState,
     validation,
     alertId,
   }: NotificationEvent) {
     const { data, headers, responseTime, status, error } = validation.response
 
-    this.httpClient
-      .post('/events', {
-        monikaId: this.monikaId,
-        alertId,
-        event: probeState === 'DOWN' ? 'incident' : 'recovery',
-        response: {
-          body: data,
-          headers: typeof headers === 'object' ? headers : {},
-          size:
-            typeof headers === 'object' ? headers['content-length'] : undefined,
-          status, // status is http status code
-          time: responseTime,
-          error,
-        },
-      })
-      .then(() => {
-        log.info(
-          `[Symon] Send ${
-            probeState === 'DOWN' ? 'incident' : 'recovery'
-          } event for Alert ID: ${alertId} succeed`
-        )
-      })
-      .catch((error) => {
-        log.error(
-          `[Symon] Send ${
-            probeState === 'DOWN' ? 'incident' : 'recovery'
-          } event for Alert ID: ${alertId} failed.  ${(error as Error).message}`
-        )
+    try {
+      const resp = await sendHttpRequestFetch({
```

Comment:
I guess this API call is missing `x-api-key` header ?

maybe we can create a reusable function inside symon client, something like
```
private fetcher() {
  sendHttpRequestFetch({
    // base url
    // common headers
  })
}
```